src/Common/Handlers/PublishPackageHandler.cs

```diff
@@ -53,11 +53,11 @@ private async Task UploadPackage(string filePath, string datagroup)
                 {
                     // publish
                     Session.LogDebug("Uploading package...");
-                    var publishNewNewStreamingOutput = await new PackageManagement.PublishApplicationPackageStreamingInput
+                    var publishNewNewStreamingOutput = await new PackageManagement.PublishApplicationPackageBaseStreamingInput
```

Comment:
did you validate this?

Reply:
Nop... But does not exist any other on the LBOs from what I saw.

Reply:
![image](https://github.com/criticalmanufacturing/portal-sdk/assets/22283190/0d76727e-7df1-41ba-9c41-9a9a44487ac3)

Tested locally and seems to work